packages/augur-sdk/src/api/ContractEvents.test.ts

```diff
@@ -85,8 +85,48 @@ test('get event topics', async () => {
   const eventTopic = 'foobarington';
   const provider = makeProviderMock({ eventTopic });
 
-  const contractEvents = new ContractEvents(provider, '0x0', '0x0', '0x0', '0x0');
+  const contractEvents = new ContractEvents(provider, '0x0there', '0x0', '0x0', '0x0');
```

Comment:
0x0there ?

Reply:
Not needed in this test, actually. The mocks use a totally fake address of `0xthere` --- See line 70 of the same file.